samples/Network/enforce-firewall-route-table-on-subnet/azurepolicy.json

```diff
@@ -0,0 +1,55 @@
+{
+	"type": "Microsoft.Authorization/policyDefinitions",
+	"name": "enforce-firewallroutetable-on-subnet",
+	"properties": {
+		"displayName": "Route table of every subnet",
+		"description": "This policy enforces a specific Route Table on every subnet",
+		"parameters": {
+			"rtId": {
+				"type": "String",
+				"metadata": {
+					"description": "Resource Id of the Firewall route table",
+					"displayName": "Firewall route table Id",
+					"strongType": "RouteTable"
+				}
+			}
+		},
+		"policyRule": {
+			"if": {
+				"anyOf": [
+					{
+						"allOf": [
+							{
+								"field": "type",
+								"equals": "Microsoft.Network/virtualNetworks"
+							},
+							{
+								"not": {
+									"field": "Microsoft.Network/virtualNetworks/subnets[*].routeTable.id",
+									"equals": "[parameters('rtId')]"
```

Comment:
just FYI for the future policy supports a "notEquals" operator which makes negations like this simpler to read